### ISSUE TITLE
not unlocking if other NFC app is open in background (open when locked) e.g. the nxp taginfo app

### ISSUE THREAD

**Issue by Lafunamor** [mclear/Android_NFC_Ring_Unlock#78]
if I have the NXP tag info app open then lock my screen and then try to unlock the device the unlock app will not receive the NFC information. as a workaround you can open the audio settings by pushing a volume button then click on the settings icon

**Comment by linxcat** [comment]
I have this problem as well, Right now the solution is "press home" and in order to do that you must not use "use without lockscreen" in the settings menu. I'm not really sure what the purpose of the "no lockscreen" function is at this point

**Comment by JohnMcLear** [comment]
Confirmed.

@r2DoesInc any thoughts on this?

Afaik the issue is that the OS is allowing the foreground app to take exclusive control of NFC which is the desired behavior however in this instance it's catching us out.  This is probably a wontfix due to the way we have to implement the app.  We "might" be able to exit the control app on home button press I guess?

**Comment by Lokki-P** [comment]
Good housekeeping behaviour is a user necessity which negates this issue.

**Comment by r2DoesInc** [comment]
Agreed lokki. When the other is active when you lock the phone, they give the nfc action to that app. Pressing home or back will close that app - even if you cant visible see it - and you should be able to scan then.
